src/pages/home/report/ReportFooter.js

```diff
@@ -57,53 +57,54 @@ const defaultProps = {
     shouldDisableCompose: false,
 };
 
-class ReportFooter extends React.Component {
+const ReportFooter = ({report, reportActions, isOffline, onSubmitComment, errors, pendingAction, shouldShowComposeInput, shouldDisableCompose, isSmallScreenWidth, isComposerFullSize}) => {
```

Comment:
The function should accept only one parameter and that is `props`.

Reply:
https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#should-i-declare-my-components-with-arrow-functions-const-or-the-function-keyword

we should use function instead of const